### PR TITLE
added the other  mapper and tests

### PR DESCRIPTION
**Pull request by Hussiendev**

### CODE REVIEW

File: tests/Toymaper.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    expect(toy.isBatteriesRequired()).toBe(true);
+    expect(toy.isEducational()).toBe(false);
+  });
+
+  it("parses numeric ageGroup from string input", () => {
+    const mapper = new ToyMapper();
+    const input = ["2", "Building Set", "10", "LEGO", "Plastic", "false", "true"];
+    const toy = mapper.map(input);
+
+    expect(toy.getAgeGroup()).toBe(10);
```
Comment (samish-dev) [inline tests/Toymaper.ts:33]: nice test coverage and handling edge cases

**Review by samish-dev — APPROVED** [review]
